moving_nest/fv_moving_nest_types.F90

```diff
@@ -42,212 +40,229 @@ module fv_moving_nest_types_mod
 
-    real, allocatable  :: parent_orog_grid(:,:)        _NULL  ! parent orography -- only used for terrain_smoother=1.
+    real, allocatable  :: parent_orog_grid(:,:)        _NULL  !> parent orography -- only used for terrain_smoother=1.
     !     raw or filtered depending on namelist option,in meters
 
     ! Soil variables
-    real, allocatable  :: deep_soil_temp_grid(:,:)     _NULL  ! deep soil temperature at 5m, in degrees K
-    real, allocatable  :: soil_type_grid(:,:)          _NULL  ! STATSGO soil type
+    real, allocatable  :: deep_soil_temp_grid(:,:)     _NULL  !> deep soil temperature at 5m, in degrees K
+    real, allocatable  :: soil_type_grid(:,:)          _NULL  !> STATSGO soil type
 
     ! Vegetation variables
-    real, allocatable  :: veg_frac_grid(:,:)           _NULL  ! vegetation fraction
-    real, allocatable  :: veg_type_grid(:,:)           _NULL  ! IGBP vegetation type
-    real, allocatable  :: veg_greenness_grid(:,:)      _NULL  ! NESDIS vegetation greenness; netCDF file has monthly values
+    real, allocatable  :: veg_frac_grid(:,:)           _NULL  !> vegetation fraction
+    real, allocatable  :: veg_type_grid(:,:)           _NULL  !> IGBP vegetation type
+    real, allocatable  :: veg_greenness_grid(:,:)      _NULL  !> NESDIS vegetation greenness; netCDF file has monthly values
 
     ! Orography variables
-    real, allocatable  :: slope_type_grid(:,:)         _NULL  ! legacy 1 degree GFS slope type
+    real, allocatable  :: slope_type_grid(:,:)         _NULL  !> legacy 1 degree GFS slope type
 
     ! Albedo variables
-    real, allocatable  :: max_snow_alb_grid(:,:)       _NULL  ! max snow albedo
-    real, allocatable  :: facsf_grid(:,:)              _NULL  ! fractional coverage with strong cosz dependency
-    real, allocatable  :: facwf_grid(:,:)              _NULL  ! fractional coverage with weak cosz dependency
+    real, allocatable  :: max_snow_alb_grid(:,:)       _NULL  !> max snow albedo
+    real, allocatable  :: facsf_grid(:,:)              _NULL  !> fractional coverage with strong cosz dependency
+    real, allocatable  :: facwf_grid(:,:)              _NULL  !> fractional coverage with weak cosz dependency
 
     ! Snow free albedo
     !   strong cosz angle dependence = black sky
     !   weak cosz angle dependence = white sky
     !  From the chgres code in static_data.F90, we see the linkage of variable names:
-    !   type(esmf_field), public           :: alvsf_target_grid !< visible black sky albedo
-    !   type(esmf_field), public           :: alvwf_target_grid !< visible white sky albedo
-    !   type(esmf_field), public           :: alnsf_target_grid !< near ir black sky albedo
-    !   type(esmf_field), public           :: alnwf_target_grid !< near ir white sky albedo
+    !   type(esmf_field), public           :: alvsf_target_grid !> visible black sky albedo
```

Comment:
So these were using the correct "!<" and you changed them to "!>" which will cause the comment to apply to the next variable, not the variable you intend.